notebooks/tensorflow/getting-started-keras.ipynb

```diff
@@ -3814,11 +3810,16 @@
       "cell_type": "code",
       "source": [
         "# Setup Learning Rate decay.\n",
-        "lr_decay = tf.keras.callbacks.LearningRateScheduler(\n",
+        "lr_decay_cb = tf.keras.callbacks.LearningRateScheduler(\n",
         "    lambda epoch: LEARNING_RATE + 0.02 * (0.5 ** (1 + epoch)),\n",
-        "    verbose=True)"
+        "    verbose=True)\n",
+        "\n",
+        "# Setup TensorBoard callback.\n",
+        "tensorboard_cb = tf.keras.callbacks.TensorBoard(\n",
+        "      os.path.join(args.job_dir, 'keras_tensorboard'),\n",
```

Comment:
I don't think this works, because when running in the notebook the `args` variable is not actually defined (since we're not parsing command-line flags).

You could either export locally (`keras_tensorboard`) or to the Cloud Storage dir that the user has already set up (`JOB_DIR + '/keras_tensorboard'`) - compare to both options in the  ["Export the model for serving" section](https://colab.sandbox.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/master/notebooks/tensorflow/getting-started-keras.ipynb#scrollTo=gmTiOQObmTqd). Depends on how you want to use TensorBoard (see overall comment).